### ISSUE TITLE
pygtksourceview/gtksourceview2 test failure

### ISSUE THREAD

**Issue by DomT4** [Homebrew/homebrew#40824]
Just putting this here for tracking. After a full `brew upgrade` following the GTK+ changes:

```
Testing pygtksourceview
==> python -c import gtksourceview2
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: could not import gtk
Error: pygtksourceview: failed
Failed executing: python -c import\ gtksourceview2
```

```
~> python -c "import gtk"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/local/lib/python2.7/site-packages/gtk-2.0/gtk/__init__.py", line 40, in <module>
    from gtk import _gtk
  File "/usr/local/lib/python2.7/site-packages/cairo/__init__.py", line 1, in <module>
    from _cairo import *
ImportError: dlopen(/usr/local/lib/python2.7/site-packages/cairo/_cairo.so, 2): Symbol not found: _cairo_xlib_surface_get_depth
  Referenced from: /usr/local/lib/python2.7/site-packages/cairo/_cairo.so
  Expected in: /usr/local/lib/libcairo.2.dylib
 in /usr/local/lib/python2.7/site-packages/cairo/_cairo.so
```

Will look at it once the GTK+ situation is stabilised.

**Comment by tschoonj** [comment]
Is definitely related to it. We will have to revision bump py2cairo and py3cairo to solve it

**Comment by DomT4** [comment]
Yeah, that'll solve it. Just waiting on the CI to be less backed up before piling on another PR. Issues don't trigger the CI so it's a relatively safe place to leave to-do notes, heh.
